src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -60,7 +61,128 @@ enum DruidSpells
     SPELL_DRUID_BALANCE_T10_BONUS           = 70718,
     SPELL_DRUID_BALANCE_T10_BONUS_PROC      = 70721,
     SPELL_DRUID_SUNFIRE_DAMAGE              = 164815,
-    SPELL_DRUID_SURVIVAL_INSTINCTS          = 50322
+    SPELL_DRUID_SURVIVAL_INSTINCTS          = 50322,
+    SPELL_DRUID_TRAVEL_FORM                 = 783
+};
+
+enum DruidForms
+{
+    DRUID_AQUATIC_FORM                      = 1066,
+    DRUID_FLIGHT_FORM                       = 33943,
+    DRUID_SWIFT_FLIGHT_FORM                 = 40120,
+    DRUID_STAG_FORM                         = 165961
+};
+
+// 783 - Travel Form
+class spell_dru_travel_form : public SpellScriptLoader
+{
```

Comment:
The same for this, try ordering alphabetically by spell_script_name as the header of the file says